### PR TITLE
no requests version of the app

### PR DESCRIPTION
**Pull request by ocke**
:v:

/cc @zendesk/vegemite

### Description
no more proxy needed!

### References
* JIRA: 

### Risks
* (low) chuck norris stops working

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,80 +1,68 @@
...
-        renderNorris: function (fact) {
-            var image = 'norris-' + this.counter + '.jpg';
-            this.counter = Math.max(1,((this.counter + 1) % 4));
-            this.switchTo('norris', {
-                fact: fact,
-                image: image
-            });
-            this.timeout = _.delay(this.ajax.bind(this, 'fetchNorris'), 10000);
-        },
+      joke.joke = joke.joke.replace(/Chuck Norris/g, this.currentUser().name());
```
Comment (xuebingli) [inline app.js]: `joke.joke`?

**Comment by sandlerr** [comment]
:+1:

File: app.js (hunk 1 above)
Comment (ocke) [inline app.js]: xuebing.joke

Comment (ocke) [inline app.js]: joke = { id: 1, categories: [], joke: "some joke" }

Do you have a better name for the variable?

Comment (xuebingli) [inline app.js]: How about `content`? `text`?

Comment (xuebingli) [inline app.js]: Honestly, anything other than `joke.joke` would be better

**Comment by xuebingli** [comment]
👍
it's a chuck norris app. what is the worst that can happen?
AhwuGl0MPNzyg

**Comment by ocke** [comment]
Can I include that gif in the assets? lol

File: app.js (hunk 1 above)
Comment (ocke) [inline app.js]: content.joke doesn't sound that great either

Comment (sandlerr) [inline app.js]: `joke.content`?

**Pull request merged**